src/Core/Security/OAuth2/TokenAuthenticator.php

```diff
@@ -71,58 +61,54 @@ public function start(Request $request, AuthenticationException $authException =
 
     public function supports(Request $request): bool
     {
-        try {
-            $authorization = $request->headers->get('Authorization') ?? null;
-            if (null === $authorization) {
-                return false;
-            }
-            $explode = explode(' ', $authorization);
-            if (count($explode) >= 2) {
-                $token = $explode[1];
+        $authorization = $request->headers->get('Authorization') ?? null;
+        if (null === $authorization) {
+            return false;
+        }
+        $explode = explode(' ', $authorization);
+        if (count($explode) >= 2) {
```

Comment:
It was already wrong initially, but there should be an `else` statement that returns false if there are not two parts